docs/_data/sidebar.yml

```diff
@@ -117,6 +118,22 @@
     en: Aura
     jp: Aura
   baseUrl: /aura/
+=======
+    - title: Writing
```

Comment:
This is showing a merge conflict. From reviewing the head vs. the changes that were causing the merge conflict, it looks like we want to keep the head. That being said, can the changes in this file be removed?